### PR TITLE
Update issue form to cover other types of support besides projects

### PR DESCRIPTION
**Pull request by vincerubinetti**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/issue.yaml [hunk 1]
```diff
@@ -31,8 +33,41 @@ body:
...
+
+  - type: dropdown
+    id: category
+    attributes:
+      label: Type of support
+      description: Which of these best describe the type of support you need?
+      options:
+        - Short/medium-term project (ongoing)
+        - Debugging/troubleshooting (one-off)
+        - Consulting/education (one-off)
```
Comment (d33bs) [inline .github/ISSUE_TEMPLATE/issue.yaml:49]: Are all of the options a form of consulting? If so, maybe removing `consulting` here could help clarify.

File: .github/ISSUE_TEMPLATE/issue.yaml [hunk 2]
```diff
@@ -31,8 +33,41 @@ body:
...
+      required: true
+
+  - type: dropdown
+    id: category
+    attributes:
+      label: Type of support
+      description: Which of these best describe the type of support you need?
+      options:
+        - Short/medium-term project (ongoing)
+        - Debugging/troubleshooting (one-off)
```
Comment (d33bs) [inline .github/ISSUE_TEMPLATE/issue.yaml:48]: Suggesting the addition of `Support` here (sometimes I feel like folks might just want guidance on "am I doing this right").
```suggestion
        - Support/Debugging/troubleshooting (one-off)
```

**Review by d33bs — APPROVED** [review]
Nice job, thought these were good additions. Left a couple of minor comments.

Does the Google Form also need an update after this? Or should we drop that option?

**Comment by vincerubinetti** [comment]
> Nice job, thought these were good additions. Left a couple of minor comments.
> 
> 
> 
> Does the Google Form also need an update after this? Or should we drop that option?

Feel free to take lead on this while i'm gone and merge. I think you should have access to the form as well.

**Comment by d33bs** [comment]
Thanks @vincerubinetti , good luck with things this week! @falquaddoomi and @the-mayer , interested to get your input too before a merge, let me know if you have any thoughts.

**Review by the-mayer — APPROVED** [review]
These all seem like positive changes to me :thumbsup:

File: .github/ISSUE_TEMPLATE/issue.yaml [hunk 3]
```diff
@@ -31,8 +33,41 @@ body:
...
+      default: 0
+    validations:
+      required: true
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: |
+        Describe the specific support you need from SET. E.g.:
```
Comment (falquaddoomi) [inline .github/ISSUE_TEMPLATE/issue.yaml:61]: I worry that the reader might think they need to answer all the questions on this list, despite it saying "E.g.". I get that we might not get all the information we need if we soften the language, but IMHO I'd rather people submit "incomplete" requests than be put off making a request all.

Totally willing to change this, but here's a first shot:

```suggestion
        Describe the support you need from SET. Feel free to put what you need in your own words, but here are some common questions:
```

**Review by falquaddoomi — COMMENTED** [review]
Looks good overall! I made a few comments.

I like that the description has been simplified; the items that were removed are things we should be discussing in a requirements-gathering session with the collaborator anyway, IMHO, not things we need upfront.

**Review by falquaddoomi — APPROVED** [review]

**Pull request merged**